suite/suite.go

```diff
@@ -63,6 +63,20 @@ func failOnPanic(t *testing.T) {
 	}
 }
 
+// Run provides suite functionality around golang subtests.  It should be
+// called in place of t.Run(name, func(t *testing.T)) in test suite code
+// to expose setup and teardown functionality for each subtest.  The passed-in
+// func will be executed as a subtest with a fresh instance of t.  Provides
+// compatibility with go test pkg -run TestSuite/TestName/SubTestName.
+func (suite *Suite) Run(name string, subtest func()) {
```

Comment:
Any reason not to return a `bool` similar to the return value of `T.Run`?

Reply:
Nope, I didn't realize `t.Run` returned a bool.  Updated to expose that return value.